### ISSUE TITLE
Command for telling who's on telnet

### ISSUE THREAD

**Issue by zD12** [TotalFreedom/BukkitTelnet#8]
Hi.
Is it possible to create a command that can tell who's on telnet?

**Comment by WilliePlaysMC** [comment]
No. This defines the point of telnet. Telnet is for admins to hiddenly watch others.

**Comment by JeromSar** [comment]
@WilliePlaysMC That's not really true, admins can see who connects with telnet through the logviewer.

**Comment by Impolse** [comment]
I think that if you add that command, it would only be accessible to administrators.
